frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue

```diff
@@ -475,7 +475,7 @@ const tableContainer = computed(() => {
 <style module lang="scss">
 .table {
   scroll-behavior: smooth;
-  max-height: calc(100vh - 310px);
+  max-height: calc(100vh - 340px);
   overflow: auto;
 }
```

Comment:
I see 3 different components having the exact same CSS class and the same change, maybe it would make more sense if it was extracted to an scss file and imported.